mne_g3d/_utils.py

```diff
@@ -40,3 +43,80 @@ def offset_hemi(vertices, hemi, offset=0.0):
         vertices[:, 0] -= (np.min(vertices[:, 0]) + offset)
 
     return vertices
+
+
+def get_mesh_cmap(act_data, cmap_str='auto'):
```

Comment:
This seems almost entirely redundant with `_limits_to_control_points`, it's better not to create functions like this unless necessary